pliers/extractors/text.py

```diff
@@ -313,3 +334,52 @@ def _extract(self, stim):
         features = ['sentiment_' + k for k in scores.keys()]
         return ExtractorResult([list(scores.values())], stim, self,
                                features=features)
+
+class SpaCyExtractor(TextExtractor):
+
+    
+    def __init__(self, model='en_core_web_sm', features=None):
+        verify_dependencies(['spacy'])
+        try: 
+            print(model)
+            model_na=model[:2]
+            self.model = spacy.load(model_na)
+            self.features= features
+            print('loaded model: ',self.model )
+        except (ImportError, IOError, OSError) as e:
+            logging.error("Spacy Models ('"+model+"') not found. Downloading and installing")
+            #spacy_download.download_model("en_core_web_sm-2.0.0/en_core_web_sm-2.0.0.tar.gz")
+            os.system('python -m spacy download '+model)
+            
+        super(SpaCyExtractor, self).__init__()
+           
+            
+            
+    def _extract(self, stim):
+        
+        inputTxt=self.model(stim.text)
```

Comment:
Per Python naming conventions, variable names should use snake case (i.e., `input_txt`).

Reply:
Note: `input_txt`, not `input_Txt`.